src/hotspot/share/jvmci/jvmciCodeInstaller.cpp

```diff
@@ -760,6 +770,17 @@ JVMCI::CodeInstallResult CodeInstaller::install(JVMCICompiler* compiler,
         nm->maybe_print_nmethod(directive);
         DirectivesStack::release(directive);
       }
+
+      if (nm != nullptr) {
+        if (_nmethod_entry_patch_offset != -1) {
+          FormatBuffer<> msg("%s", "");
+          BarrierSetNMethod* bs_nm = BarrierSet::barrier_set()->barrier_set_nmethod();
+
+          if (!bs_nm->verify_barrier(nm, msg)) {
+            JVMCI_THROW_MSG_(IllegalArgumentException, err_msg("nmethod entry barrier is malformed: %s", msg.buffer()), JVMCI::ok);
+          }
+        }
+      }
```

Comment:
We have this typedef:
```
typedef FormatBuffer<> err_msg;

```
So, it looks weird to have code that mixes FormatBuffer<> and err_msg.

Reply:
I converted all bare usages of FormatBuffer<> to err_msg which seems to be the general usage.